### PR TITLE
Readmefr

### PR DESCRIPTION
**Pull request by valasulth**
# Description

## Corresponding Issue

# Screenshots

---

Reviewing this pull request? Check out our [Code Review Practices](https://github.com/ifmeorg/ifme/wiki/Code-Review-Practices) guide if you haven't already!

### CODE REVIEW

File: README-FR.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+# if me
+
+  
+
```
Comment (julianguyen) [inline README-FR.md]: Let's remove the unnecessary white-space and fix the grammar.

File: README-FR.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
+# if me
```
Comment (julianguyen) [inline README-FR.md]: Each README file has a header section with the badges and links to the other translations:

image

You'll want to add this and also add a link to this file in the other READMEs.

**Comment by julianguyen** [comment]
Since it looks like you're a French speaker, can you review @nicovak?

File: README-FR.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+  
+
+[if me](https://www.if-me.org/) est une communautée  sur les expériences sur la santé mentale qui encourage les gens a partager leur histoire personnel avec des personnes de confiance.
+les gens de confiance sont ceux avec qui on parle toues les jours , les membres de la famille, les amis, les professeurs, , et ceux qui travaille pour les expériences sur la santé.
```
Comment (julianguyen) [inline README-FR.md]: Should be capitalized to `Les`.

Comment (julianguyen) [reply README-FR.md]: Remove extra comma in `, et ceux qui travaille pour les expériences sur la santé`.

File: README-FR.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+Regarder notre [Wiki](https://github.com/ifmeorg/ifme/wiki) pour avoir la documentation.
+
+  
+
+### Installation
+
+  
+
+les informations pour installer et configurer l'application [ici](https://github.com/ifmeorg/ifme/wiki/Installation). Les instances de test, de développement et de production sont couvertes.
```
Comment (julianguyen) [inline README-FR.md]: Should be capitalized to `Les`.

File: README-FR.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+Nous vous encourageons [a vous mettre](https://github.com/ifmeorg/ifme/wiki/Contributor-Blurb) sur la page des contributeurs.
+
+  
+
+## Donation
+
+  
+
+We also welcome financial contributions in full transparency on our
```
Comment (julianguyen) [inline README-FR.md]: You can remove this line.

File: README-FR.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+On accepte aussi les contributions financières en toute transparence sur notre page
+[Open Collective](https://opencollective.com/ifme).
+
+Tout le modèle peut participer aux frais. Si la dépense a un sens pour le développement de  
+  
+la communauté, il sera "fusionné" dans le livre de notre Open Collective par le  
+  
+Les contributeurs principaux et les personnes qui ont effectué des dons seront remboursés.
```
Comment (julianguyen) [inline README-FR.md]: It should be `par les contributeurs principaux et les personnes qui ont effectué des dons seront remboursés.`

File: README-FR.md [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+
+<a  href="https://opencollective.com/ifme/sponsor/9/website"  target="_blank"><img  src="https://opencollective.com/ifme/sponsor/9/avatar.svg"></a>
+
+  
+
+## License
+
+  
+
+Le Code source est sous licence GNU AGPLv3. Pour plus d'informations allez voir
```
Comment (julianguyen) [inline README-FR.md]: I don't think code should be capitalized.

File: README-FR.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
+# if me
+
+[![CircleCI](https://circleci.com/gh/ifmeorg/ifme/tree/master.svg?style=svg)](https://circleci.com/gh/ifmeorg/ifme/tree/master)
+[![Code Climate](https://codeclimate.com/github/ifmeorg/ifme/badges/gpa.svg)](https://codeclimate.com/github/ifmeorg/ifme)
+[![Test Coverage](https://api.codeclimate.com/v1/badges/f9444a4d4116720518fe/test_coverage)](https://codeclimate.com/github/ifmeorg/ifme/test_coverage)
+[![Backers on Open Collective](https://opencollective.com/ifme/backers/badge.svg)](#backers)
+[![Sponsors on Open Collective](https://opencollective.com/ifme/sponsors/badge.svg)](#sponsors)
+
+ Readme en [anglais](https://github.com/ifmeorg/ifme/blob/master/README.md), [portugais](https://github.com/ifmeorg/ifme/blob/master/README-PT.md), [espagnol](https://github.com/ifmeorg/ifme/blob/master/README-ES.md)
```
Comment (julianguyen) [inline README-FR.md]: Can you capitalize Anglais, Portugais, and Espagnol? You will also have to add French to the README.md, README-PT.md, and README-ES.md.

File: README-FR.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+
+[if me](https://www.if-me.org/) est une communautée  sur les expériences sur la santé mentale qui encourage les gens a partager leur histoire personnel avec des personnes de confiance.
+les gens de confiance sont ceux avec qui on parle toues les jours , les membres de la famille, les amis, les professeurs,et ceux qui travaille pour les expériences sur la santé.
+
+
```
Comment (julianguyen) [inline README-FR.md]: Can you remove these double line breaks through the doc?

File: README-FR.md [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+ Readme en [anglais](https://github.com/ifmeorg/ifme/blob/master/README.md), [portugais](https://github.com/ifmeorg/ifme/blob/master/README-PT.md), [espagnol](https://github.com/ifmeorg/ifme/blob/master/README-ES.md)
+
+[if me](https://www.if-me.org/) est une communautée  sur les expériences sur la santé mentale qui encourage les gens a partager leur histoire personnel avec des personnes de confiance.
+les gens de confiance sont ceux avec qui on parle toues les jours , les membres de la famille, les amis, les professeurs,et ceux qui travaille pour les expériences sur la santé.
+
+ 
+Traiter avec la santé mentale c'est ce qui fait de nous des humains. Mais pour beaucoup de nous, c'est un combat d'être ouvert à ce sujet. Pas tout le monde est un
+ conseiller ou un thérapeute
+les personnes avec lesquelles nous interagissons quotidiennement façonnent nos émotions et notre comportement.
```
Comment (julianguyen) [inline README-FR.md]: Lines 16 and 17 can be a part of line 15.

File: README-FR.md [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+
+Le site  peut être trouvé à [if-me.org](https://www.if-me.org/).Les systèmes de conception en direct peuvent être trouvés à [design.if-me.org](http://design.if-me.org/).
+
+  
+
+On utilise le merveilleux[Contributor Covenant](http://contributor-covenant.org) pour notre code de conduite .  s'il vous plait
+
+[Lisez le](https://github.com/ifmeorg/ifme/blob/master/code_of_conduct.md)
+
+avant de rejoindre notre projet.
```
Comment (julianguyen) [inline README-FR.md]: These lines can be a part of line 25.

File: README-FR.md [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+
+  
+
+**Lisez à propos des objectifs de notre projet et comment y contribuer (not just as a developer) [ici](https://github.com/ifmeorg/ifme/blob/master/CONTRIBUTING.md).**
+
+
```
Comment (julianguyen) [inline README-FR.md]: There should just be one line break. This applies to other similar areas.

**Comment by valasulth** [comment]
thank you for helping me to do :)

**Pull request merged**